go/gramework/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.12.7 AS buildenv
+
+ENV GO111MODULE=on
+ENV CGO_ENABLED=0
+
+WORKDIR /go/src/app
+ADD . ./
+
+RUN go get
+RUN go build -a -o /go/bin/app ./
+
+FROM scratch
+COPY --from=buildenv /go/bin/app /go/bin/app
+CMD ["/go/bin/app"]
```

Comment:
Please, add ENV=production. By default, gramework logs too much for production usage, but enough for development convenience

Reply:
ok